integration/integration.go

```diff
@@ -91,6 +91,9 @@ type Options struct {
 
 	// Network is the VM's network.
 	Network *qemu.Network
+
+	// Extra environment variables to set when building (used by u-bmc)
+	ExtraBuildEnv []string
```

Comment:
Unused? Also, what's this for? Options.Env should have all Golang env variables you can set.

Reply:
Oh it's for make. Hm.

Reply:
I'm usually not a fan of broken abstractions like this, but I guess it's a test framework. Go for it?